pom.xml

```diff
@@ -89,6 +91,11 @@
                 <artifactId>commons-io</artifactId>
                 <version>${commons.io.version}</version>
             </dependency>
+            <dependency>
+                <groupId>com.fasterxml.jackson.core</groupId>
+                <artifactId>jackson-databind</artifactId>
+                <version>${jackson.databind.version}</version>
+            </dependency>
```

Comment:
Why did you add that ?

Reply:
when i thought i could skip using the CSE crac importer and use raw xml. I forgot to remove it